src/CommandLineParser.ts

```diff
@@ -54,6 +54,13 @@ export const argv = yargs.options({
         type: 'boolean',
         default: false,
         demandOption: false
+    },
+	noDeleteOnError: {
+	alias: 'd',
```

Comment:
Can you make this `e` instead? I'm sure we're going to need `d` for other purposes soon.